armi/reactor/zones.py

```diff
@@ -32,6 +32,18 @@ class Zone:
     .. impl:: A user can define a collection of armi locations.
         :id: I_ARMI_ZONE
         :implements: R_ARMI_ZONE
+
+        The Zone class facilitates the creation of a Zone object representing a
+        collection of locations in the Core. A Zone contains a group of locations
+        in the Core, used to subdivide it for analysis. Each location represents
+        an Assembly or a Block, where a single Zone must contain items of the same
+        type (i.e., Assembly or Block). Class methods are provided to add or remove
+        one or more locations to/from the Zone, and similarly, add or remove one or
+        more items with a Core location (i.e., Assemblies or Blocks) to/from the
+        Zone. In addition, several Class methods are provided to facilitate the
```

Comment:
```suggestion
        Zone. In addition, several methods are provided to facilitate the
```

Reply:
This isn't my PR, but in Python "class method" is the same as "method".

In other languages, "function" and "method" might mean different things.  But in Python a "method" is a function that is part of a class.  And that's just how Python defines those two words.